graphql_compiler/compiler/helpers.py

```diff
@@ -271,6 +275,12 @@ def __init__(self, base_location, relative_position):
         self.base_location = base_location
         self.relative_position = relative_position
 
+    def get_location_name(self):
+        """Return a unique name for the FoldScopeLocation."""
+        edge_direction, edge_name = self.relative_position
+        return (self.base_location.get_location_name()[0] + u'___' +
```

Comment:
Let's make this string concatenation a little safer some time in the future. Perhaps we can have a helper method to concatenate strings, and we can _hardcode_ unicode strings like : `u'___'` and `u'__'`, since it's easy to make mistakes here.

Reply:
This method is the only possible way to get the name of the location, and everything calls it. Since you can't construct it any other way, I really think it's about as safe as it can ever be.